vars/go.groovy

```diff
@@ -8,7 +8,11 @@ def ccv() {
         cd temp
         git clone https://${GITHUB_CREDS_PSW}@github.com/helxplatform/${REPO_NAME}.git > /dev/null
         cd ${REPO_NAME}
-        git switch master > /dev/null
+        git switch ${BRANCH_NAME} > /dev/null
```

Comment:
isn't `${BRANCH_NAME}` equal to the branch that the code is currently on, not either `master` or `main`? I thought we wanted to switch to either `master` or `main` if that fails

Reply:
Everywhere CCV is used, it's inside an "if" statement checking for master or master/main. So it took less code to use BRANCH_NAME, which will always be master or main in those cases. I can go back and add a branch test if you prefer, but then we're checking it twice. Let me know which you prefer.

Reply:
I think I do prefer the extra check, just because 1) it's more clear when reading it what it's doing and 2) it makes this function more pure. I always prefer not to rely on side effects happening before the function runs for the function to run properly, just on principle, in case we ever decide to use this function outside its current context.